iib/workers/tasks/build.py

```diff
@@ -358,6 +391,22 @@ def _opm_index_add(
         ``overwrite_from_index``. The format of the token must be in the format "user:password".
     :raises IIBError: if the ``opm index add`` command fails.
     """
+    index_image_bundles = _serve_index_db(from_index, base_dir)
+    present_bundle_hashes = [x["bundlePath"].split('@sha256:')[-1] for x in index_image_bundles]
+    bundle_hashes = [(x, x.split('@sha256:')[-1]) for x in bundles]
+    for b in present_bundle_hashes:
```

Comment:
Could you please use `bundle_hash` instead of just b? I believe it would be more intuitive